tests/ceph_ansible/test_ansible_upgrade.py

```diff
@@ -213,7 +218,38 @@ def run(ceph_cluster, **kw):
             LOG.error("Failed during cephadm adopt (rc = {})".format(rc))
             return rc
 
+        LOG.info("The value for parameter build is {}".format(build))
+        if build.startswith("5.1"):
+            installer = ceph_cluster.get_nodes(role="installer")[0]
+            base_cmd = "sudo cephadm shell -- ceph"
```

Comment:
I would suggest to create `CephadmShellMixin` for these commands. It would be usefull in other places as well

Reply:
Sure I think this could be implemented as a different PR, since it needs changes in more than one module. 
@psathyan Let me know if this makes sense to do I will create a github issue for the same for now and work on it later.

Reply:
This code would eventually go away hence I am not of the opinion of writing a separate method / object .

Reply:
Okay so for now I will retain it as it is.